hasher-matcher-actioner/terraform/main.tf

```diff
@@ -309,3 +307,5 @@ resource "aws_secretsmanager_secret_version" "te_api_token" {
   secret_id     = aws_secretsmanager_secret.te_api_token.id
   secret_string = var.te_api_token
 }
+
```

Comment:
super nit/aside: do we want to have a standard auto formatter for tf files, I've noticed some inconsistencies (beside the whitespace in this PR) that could mess with the blame...